ci/run_clang_tidy.sh

```diff
@@ -37,8 +37,15 @@ function exclude_chromium_url() {
   grep -v source/common/chromium_url/
 }
 
+# Do not run clang-tidy against win32 impl
+# TODO: We should run clang-tidy against win32 impl. But currently we only have 
+# linux ci box.
+function exclude_win32_impl() {
+  grep -v source/common/filesystem/win32/
+}
```

Comment:
Disable the clang-tidy on win32 implementation. Since linux platform does not have some windows special headers (windows.h or io.h)

Reply:
Can you please split this out into a separate PR?